src/routes/connect/ui/wallets/PrivateKey.ts

```diff
@@ -22,10 +22,9 @@ export default class PrivateKey {
         }
     }
 
-    signTx(tx: Transaction, secretKey: Keypair): string {
+    signTx(tx: Transaction, secretKey: Keypair): void {
         tx.sign(secretKey);
         const signedXDR = tx.toXDR();
         sendMessage(signedXDR);
-        return signedXDR;
```

Comment:
This change is reverted later, now it returns the value, but it's the component is in charge to send the message now